protocol/communities/manager.go

```diff
@@ -73,36 +73,36 @@ var (
 )
 
 type Manager struct {
-	persistence                      *Persistence
-	encryptor                        *encryption.Protocol
-	ensSubscription                  chan []*ens.VerificationRecord
-	subscriptions                    []chan *Subscription
-	ensVerifier                      *ens.Verifier
-	ownerVerifier                    OwnerVerifier
-	identity                         *ecdsa.PrivateKey
-	installationID                   string
-	accountsManager                  account.Manager
-	tokenManager                     TokenManager
-	collectiblesManager              CollectiblesManager
-	logger                           *zap.Logger
-	stdoutLogger                     *zap.Logger
-	transport                        *transport.Transport
-	timesource                       common.TimeSource
-	quit                             chan struct{}
-	torrentConfig                    *params.TorrentConfig
-	torrentClient                    *torrent.Client
-	walletConfig                     *params.WalletConfig
-	communityTokensService           communitytokens.ServiceInterface
-	historyArchiveTasksWaitGroup     sync.WaitGroup
-	historyArchiveTasks              sync.Map // stores `chan struct{}`
-	periodicMembersReevaluationTasks sync.Map // stores `chan struct{}`
-	torrentTasks                     map[string]metainfo.Hash
-	historyArchiveDownloadTasks      map[string]*HistoryArchiveDownloadTask
-	stopped                          bool
-	RekeyInterval                    time.Duration
-	PermissionChecker                PermissionChecker
-	keyDistributor                   KeyDistributor
-	communityLock                    *CommunityLock
+	persistence                  *Persistence
+	encryptor                    *encryption.Protocol
+	ensSubscription              chan []*ens.VerificationRecord
+	subscriptions                []chan *Subscription
+	ensVerifier                  *ens.Verifier
+	ownerVerifier                OwnerVerifier
+	identity                     *ecdsa.PrivateKey
+	installationID               string
+	accountsManager              account.Manager
+	tokenManager                 TokenManager
+	collectiblesManager          CollectiblesManager
+	logger                       *zap.Logger
+	stdoutLogger                 *zap.Logger
+	transport                    *transport.Transport
+	timesource                   common.TimeSource
+	quit                         chan struct{}
+	torrentConfig                *params.TorrentConfig
+	torrentClient                *torrent.Client
+	walletConfig                 *params.WalletConfig
+	communityTokensService       communitytokens.ServiceInterface
+	historyArchiveTasksWaitGroup sync.WaitGroup
+	historyArchiveTasks          sync.Map // stores `chan struct{}`
+	membersReevaluationTasks     sync.Map // stores `membersReevaluationTask`
+	torrentTasks                 map[string]metainfo.Hash
+	historyArchiveDownloadTasks  map[string]*HistoryArchiveDownloadTask
```

Comment:
If we are expecting it to grow up, maybe it makes sense to do a refactoring and create a `TasksManager`.

Reply:
I totally agree. I didn't introduce a new one, just changed the name. Will keep it on my radar :+1: